mcs/build/tests.make

```diff
@@ -96,8 +96,10 @@ endif
 
 test_assemblies :=
 
+test_lib_output = $(topdir)/class/lib/$(PROFILE)/$(test_lib)
```

Comment:
After changing the resource includes in the Makefiles, I didn't run into any tests hit by the jenkins CI script that had any issues with the movement on the fullaot profile. The resources are bundled right into the dll. We'll have to see if any failures on other profiles pop up. The relative paths can be fixed. The tests are being run from the same directories, for what it's worth.